src/ActionList2/Group.tsx

```diff
@@ -1,18 +1,51 @@
 import React from 'react'
+import {useSSRSafeId} from '@react-aria/ssr'
 import Box from '../Box'
 import {SxProp} from '../sx'
-import {Header, HeaderProps} from './Header'
-import {ListProps} from './List'
+import {ListContext, ListProps} from './List'
+import {AriaRole} from '../utils/types'
 
-export type GroupProps = HeaderProps &
-  SxProp & {
+export type GroupProps = {
+  /**
+   * Style variations. Usage is discretionary.
+   *
+   * - `"filled"` - Superimposed on a background, offset from nearby content
+   * - `"subtle"` - Relatively less offset from nearby content
+   */
+  variant?: 'subtle' | 'filled'
+  /**
+   * Primary text which names a `Group`.
+   */
+  title?: string
+  /**
+   * Secondary text which provides additional information about a `Group`.
+   */
+  auxiliaryText?: string
+  /**
+   * The ARIA role describing the function of the list inside `Group` component. `listbox` or `menu` are a common values.
+   */
+  role?: AriaRole
```

Comment:
Question: Given there's a limited number of suitable roles for this component, can we help the user decision along by providing a union of those compatible types?

Reply:
I'm a bit hesitant because it's heavily context dependent so it's hard to create good types for it. 

1. I'm hoping to set these automatically when you use our menu components like ActionMenu or SelectPanel and increase their coverage.
2. For building your menu story (common escape hatch in memex), I'm a bit hesitant about adding "hints" that send you the wrong way - classic a11y story, better to not say anything than say the wrong thing

I think I'll come around to this feeling more confident when we replace the escape hatches in memex with the right menu components (ActionMenu/SelectPanel/etc)